fields/acf-Typography-v4.php

```diff
@@ -35,17 +35,17 @@ function __construct( $settings )
 			// This makes life easy when creating the field options as you don't need to use any if( isset('') ) logic. eg:
 			'display_properties'	=> array(),
 			'required_properties'	=> array(),
-			'font_size'				=> 15,
-			'font_weight'			=> '400',
-			'font_family'			=> 'Arial, Helvetica, sans-serif',
-			'font_style'			=> 'normal',
-            'font_variant'			=> 'normal',
-            'font_stretch'			=> 'normal',
-			'text_align'			=> 'left',
-			'letter_spacing'		=> 0,
-			'text_decoration'		=> 'none',
-			'text_color'			=> '#000',
-			'text_transform'		=> 'none'
+			'font_size'		=> 15,
+			'font_weight'		=> '400',
+			'font_family'		=> 'Arial, Helvetica, sans-serif',
+			'font_style'		=> 'normal',
+                        'font_variant'		=> 'normal',
+                        'font_stretch'		=> 'normal',
+			'text_align'		=> 'left',
+			'letter_spacing'	=> 0,
+			'text_decoration'	=> 'none',
+			'text_color'		=> '#000',
+			'text_transform'	=> 'none'
```

Comment:
I don't see the indentation problem here. I don't mind fixing it, but I don't know what you want fixed. All array items line up. Previously they did not. Do you want me to change it back to where font_variant & font_stretch don't line up? or do you want me to get rid of the tabs that make the '=> VALUE' for each line to line up?

As it stands now:
```
		$this->defaults = array(
			// add default here to merge into your field. 
			// This makes life easy when creating the field options as you don't need to use any if( isset('') ) logic. eg:
			'display_properties'	=> array(),
			'required_properties'	=> array(),
			'font_size'		=> 15,
			'font_weight'		=> '400',
			'font_family'		=> 'Arial, Helvetica, sans-serif',
			'font_style'		=> 'normal',
                        'font_variant'		=> 'normal',
                        'font_stretch'		=> 'normal',
			'text_align'		=> 'left',
			'letter_spacing'	=> 0,
			'text_decoration'	=> 'none',
			'text_color'		=> '#000',
			'text_transform'	=> 'none'
		);
```

Reply:
OK, I think I see it. I had to bring it into another code editor to see what you were talking about since it displays fine in my primary editor and github. There were spaces used where tabs should have been used. I hope that's what you wanted fixed. Let me know!

Reply:
@codejp3 I looked in VSCode and Github.dev editor if it helps.

Reply:
Got it. It's a VSCode editor thing. Pulling files in and making revisions with that editor so that we're seeing the same thing.